composer.json

```diff
@@ -16,7 +16,8 @@
     "phpseclib/phpseclib": "^2.0",
     "webmozart/path-util": "^2.3",
     "doctrine/doctrine-fixtures-bundle": "^2.3",
-    "mattketmo/camel": "^1.1"
+    "mattketmo/camel": "^1.1",
+    "roave/better-reflection": "^3"
```

Comment:
Was de standaard reflection niet genoeg?

Reply:
Standard reflection was not enough since `\ReflectionClass::getMethods()` only returns the methods of that class, but not its parents, traits, ...